iron-list.html

```diff
@@ -138,8 +138,15 @@
         position: relative;
       }
 
-      #items > ::content > * {
+      :host(:not([grid])) #items > ::content > * {
         width: 100%;
+      };
+  
+      :host([grid]) #items > ::content > * {

```

Comment:
replace ` :host([grid]) #items > ::content > * ` for  `#items > ::content > * `. Even for the `!grid` case, setting margin in the item isn't allowed.